### DISCUSSION TITLE
Full Page Error

### DISCUSSION THREAD

**Discussion by wilfredomerced** [twilio-labs/paste#1515] [Q&A] [answered]
I was curious on best practices for displaying a full page error

In this scenario
1. user has clicked on a link from a confirmation email 
2. the user has to log in to confirm the link but the link has expired or could experience a call failure.  

In the past SendGrid has used similar cards but wanted to see if I should be using a different component/pattern.

![error state](https://user-images.githubusercontent.com/81593493/120351493-2b89f100-c2bd-11eb-93ba-4bf042fa54d0.jpg)

**Comment by SiTaggart** [comment, marked as answer]
Hi @wilfredomerced 

I suspect the Empty States pattern would be the best place to start. We even have an example on there of it being used as an error state in the first code sandbox on the page. 

Please let us know how you get on. 

https://paste.twilio.design/patterns/empty-states

![image](https://user-images.githubusercontent.com/368249/120408406-d58c6c00-c303-11eb-9812-9387d776d2d7.png)
